src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -30,6 +30,8 @@ public class ModelManager implements Model {
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
     private final FilteredList<Person> loggedFilteredPersons;
+    private String lastExecutedCommandType = "unknown";
+    private FilteredList<Person> foundPersonsList;
```

Comment:
Perhaps it is better to switch to the `loggedFilteredPersons` instead? @WeeeHung has created a `loggedFilteredPersons` list on `Line 32` in his previous PR #107, and you may have missed the change during a synchronization merge in a previous commit.

Reply:
`loggedFilteredPersons` refers to the currently-logged filtered persons list. `foundPersonsList` keeps track of the results of the latest `find` command, used to add the results to the logbook. I used it so `log` will only save results of `find` commands, instead of the entire list, which would happen if `filteredPersons` was used.

Reply:
This is certainly a valid way to do this. However, I would like to point out that `FilteredList` is primarily a [JavaFX Collection](https://openjfx.io/javadoc/17/javafx.base/javafx/collections/transformation/FilteredList.html), designed to interact with the UI. Since the `foundPersonsList` collection does not interact with the UI, perhaps it may be better to use an alternative solution?

For example, the `FilteredList` instances are directly backed by the `getPersonList` method in the constructor of `ModelManager` (`Lines 47 to 49`). Notice that if you track the method calls during the invocation of `execute` of the `AddCommand` and `EditCommand`, the `FilteredList` instances are never directly modified ever again - changes are only made to `AddressBook`, and `FilteredList` only gets a `Predicate` to refresh it.

I believe this can be attributed to the lazy property of the `Observable` interface, which the `FilteredList` inherits - by some black box miracle, the lazily evaluated `Observable` property lets the UI update with the latest information in `AddressBook` after inducing a state-change in the relevant `FilteredList` instance.

In other words, you could potentially get the last `find` result from the `FilteredList` instance by invoking the `getPredicate()` command on `filteredPersons`, like so:

```
public void updateLoggedFilteredPersonsOverriding() {
    loggedFilteredPersons.setPredicate(filteredPersons.getPredicate());
}

public void updateLoggedFilteredPersonsAppending() {
    loggedFilteredPersons.setPredicate(loggedFilteredPersons
            .getPredicate()
            .or(filteredPersons.getPredicate())
    );
}
```

Reply:
Discussed in real life - will merge first and refactor thereafter to address this bug.